### ISSUE TITLE
Slight problems with Fermata Control on AA in my Peugeot e208

### ISSUE THREAD

**Issue by dreimer1986** [AndreyPavlenko/Fermata#351]
After I saw that my used forever AAWireless can fix the Android 14 flaw with DevMode I happily watched a few videos and playlists WHILE PARKING IN MY GARAGE OF COURSE. I use a Google Pixel 7 Pro for that. One problem I have though. I this case with the installed Fermata Control APK I guess... Both APKs are v1.9.6.

If I select a playlist and use the controls on my steering wheel for next track it seems to switch to next video once and then just restarts the current video from start when I try to skip another video. Funny enough this behavior continues even when I try to hand control it on the car screen afterwards. Maybe I am just stupid and my problem is my own fault, but if so... what do I do wrong?

But even that current behavior is of course better than the default behavior causing the last media player to startup in background again.

**Comment by AndreyPavlenko** [comment]
Sorry, but I don't understand the problem.

**Comment by dreimer1986** [comment]
Ok, I try on short. Steering wheel next track selected -> next video -> Steering wheel next track selected again -> current video starts from beginning again -> Steering wheel next track selected again -> current video starts from beginning again etc etc. I'd expect it to always go on to the next video in playlist and not restart the current one from start. But his is what happens here.

**Comment by AndreyPavlenko** [comment]
I'm unable to reproduce. Have you tried to create a new playlist, is it still reproducible?

**Comment by dreimer1986** [comment]
I tinkered a bit more with it. It's the same when I use the GUI buttons for next track. It started with the 1st video I skipped to 2nd which worked fine, too and then skipped once more and there it starts to do mentioned strange things. I don't know if that might cause problems, but I did login and I have a Premium Account on YT. So in general I did not change anything on the settings and get this behavior. I will try around a bit more including with a new YT playlist later that day. Keep you updated.
